libcodechecker/server/api/report_server.py

```diff
@@ -1827,19 +1754,168 @@ def getMissingContentHashes(self, file_hashes):
             return list(set(file_hashes) -
                         set(map(lambda fc: fc.content_hash, q)))
 
+    def __store_source_files(self, source_root, filename2hash):
```

Comment:
If there is a file named `store_handler`, shouldn't these codes be moved there instead?

Reply:
I do not wanted to move it there so all of the database session creations are visible in the report server.

Reply:
Understandable.